easybuild/easyblocks/m/mumps.py

```diff
@@ -62,19 +61,20 @@ def configure_step(self):
 
         # select Makefile.inc template and prepare compiler specific compiler flags
         comp_fam = self.toolchain.comp_family()
-        if comp_fam == toolchain.INTELCOMP:  #@UndefinedVariable
+        if comp_fam == toolchain.INTELCOMP:  # @UndefinedVariable
             make_inc_templ = 'Makefile.INTEL.%s'
             optf = "-Dintel_ -DALLOW_NON_INIT -nofor-main"
             optl = "%s -nofor-main" % optl
-        elif comp_fam == toolchain.GCC:  #@UndefinedVariable
+        elif comp_fam == toolchain.GCC:  # @UndefinedVariable
             if LooseVersion(self.version) >= LooseVersion('5.0.0'):
                 make_inc_templ = 'Makefile.debian.%s'
             else:
                 make_inc_templ = 'Makefile.gfortran.%s'
 
             optf = "-DALLOW_NON_INIT"
         else:
-            raise EasyBuildError("Unknown compiler family, don't know to prepare for building with specified toolchain.")
+            raise EasyBuildError(
+                "Unknown compiler family, don't know to prepare for building with specified toolchain.")
```

Comment:
```suggestion
            error = "Unknown compiler family, don't know to prepare for building with specified toolchain."
            raise EasyBuildError(error)

Reply:
Done slightly differently and fixed the missing "how"